### ISSUE TITLE
It seems impossible to make a prepared statement with "limit ?"

### ISSUE THREAD

**Issue by oseval** [outworkers/phantom#616]
Something like this:
`select.all().where(_.field eqs ?).limit(?).prepare()`

**Comment by iRhysBradbury** [comment]
Have you tried:
`select.where(_.field eqs ?).limit(?).prepare()`

**Comment by alexflav23** [comment]
@oseval This is not possible at this time in phantom, but I'm not sure it's supported in Cassandra.

**Comment by oseval** [comment]
@alexflav23 it's supported:
```
val prepared = session.prepare("select * from table where field = ? limit ?")
val result = session.executeAsync(
  prepared.bind()
    .setString("field", value)
    .setInt(2, 100)
)
```

**Comment by alexflav23** [comment]
@oseval Ok that's not a problem, we will fix this.

**Comment by alexflav23** [comment]
Hi @oseval, this is now fixed in 2.1.1, which will hit Maven Central in about 2 hours. The following query is now possible:

```scala
val recipe = gen[Recipe]
val limit = 1

val query = database.recipes.select.where(_.url eqs ?).limit(?).prepare()
```
